documentation/modules/security/con-certificate-authorities.adoc

```diff
@@ -6,16 +6,16 @@
 = Certificate Authorities
 
 To support encryption, each {ProductName} component needs its own private keys and public key certificates.
-All component certificates are signed by a Certificate Authority (CA) called the _cluster CA_.
+All component certificates are signed by an internal Certificate Authority (CA) called the _cluster CA_.
 
-Similarly, each Kafka client application connecting using TLS client authentication needs private keys and certificates.
-The _clients CA_ is used to sign the certificates for the Kafka clients.
+Similarly, each Kafka client application connecting to {ProductName} using TLS client authentication needs to provide private keys and certificates.
+A second internal CA, named the _clients CA_, is used to sign certificates for the Kafka clients. 
 
 == CA certificates
 
-Each CA has a self-signed public key certificate.
+Both the cluster CA and the clients CA have a self-signed public key certificate.
 
-Kafka brokers are configured to trust certificates signed by either the clients CA or the cluster CA. Components to which clients do not need to connect, such as ZooKeeper, only trust certificates signed by the cluster CA. Client applications that perform mutual TLS authentication have to trust the certificates signed by the cluster CA.
+Kafka brokers are configured to trust certificates signed by either the clients CA or the cluster CA. Components that clients do not need to connect to, such as ZooKeeper, only trust certificates signed by the cluster CA. Client applications that perform mutual TLS authentication must trust the certificates signed by the cluster CA.
```

Comment:
"Client applications that perform mutual TLS authentication must trust the certificates signed by the cluster CA.".
Mutual TLS authentication is about having the TLS encryption (so somehow trusting the server) but even using TLS client authentication. The client applications must trust the certificates signed by the cluster CA not necessarily for mutual authentication, it's enough you want TLS encryption for having the need of trusting certificates signed by the cluster CA.

Reply:
I think I understand now. I've rephrased the information and included a link to the new concept on mutual TLS authentication.

> Unless TLS encryption for external listeners is disabled, client applications must trust certificates signed by the cluster CA. This is also true for client applications that perform [mutual TLS authentication](https://strimzi.io/docs/master/#con-mutual-tls-authentication-using-uo).#

^ Please suggest further edits if needed.